src/components/MoveButton.vue

```diff
@@ -1,23 +1,28 @@
 <template>
   <div class="move-button">
-    <button class='btn btn-secondary mr-2 mb-2 move-button' @click="handleClick" @mouseenter="hover" @mouseleave="leave" :title="button.tooltip" v-b-tooltip.html v-html="customLabel || button.label || button.command" v-if="!button.times">
+    <button v-if="!button.times && !isBoosterButton" class='btn btn-secondary mr-2 mb-2 move-button' @click="handleClick" @mouseenter="hover" @mouseleave="leave" :title="button.tooltip" v-b-tooltip.html v-html="customLabel || button.label || button.command" >
     </button>
-    <b-dropdown class='mr-2 mb-2 move-button' v-else split right :text="customLabel || button.label || button.command" @click="handleRangeClick(button.times[0])">
+    <b-dropdown v-if="button.times" class='mr-2 mb-2 move-button'  split right :text="customLabel || button.label || button.command" @click="handleRangeClick(button.times[0])">
       <b-dropdown-item v-for="i in button.times" :key="i" @click="handleRangeClick(i)">{{i}}</b-dropdown-item>
     </b-dropdown>
     <b-modal v-if="button.modal" v-model="modalShow" size="lg" @ok="handleOK" @hide="modalCancel" :title="button.title || button.label || button.command" ok-title="OK, I pick this one!">
       <div  v-html="button.modal"></div>
     </b-modal>
+    <Booster v-if="button.command && isBoosterButton" class="mb-1 mr-1" @click="handleClick" :booster="button.command" />
```

Comment:
I would keep the if/else. Just move the booster in first place: `<Booster v-if="button.booster">` and then the second change the `v-if` to `v-else-if` and don't change the dropdown.